src/frontend/Sider/Personoversikt/FrittståendeBrev/FrittståendeBrev.tsx

```diff
@@ -10,11 +15,33 @@ const FrittståendeBrev: React.FC<{ valgtStønadstype: Stønadstype; fagsakId: s
     valgtStønadstype,
     fagsakId,
 }) => {
-    const { brevmaler, brevmal, settBrevmal, malStruktur } = useBrev(
+    const { request } = useApp();
+
+    const { brevmaler, brevmal, settBrevmal, malStruktur, fil, settFil } = useBrev(
         valgtStønadstype,
         'FRITTSTAENDE'
     );
 
+    const [feilmelding, settFeilmelding] = useState<string>();
+
+    const sendBrev = () => {
+        if (fil.status === RessursStatus.SUKSESS && brevmal) {
+            request<null, { pdf: string; tittel: string }>(
+                `/api/sak/frittstaende-brev/send/${fagsakId}`,
+                'POST',
+                {
+                    pdf: fil.data,
+                    tittel: brevmal,
+                }
+            ).then((res) => {
+                // TODO: settToast() ved suksess
```

Comment:
Tenkte å legge til muligheten for å ha toast i egen PR, så da legges det til her